defold-spine/editor/src/spineguiext.clj

```diff
@@ -213,6 +225,8 @@
               geom/empty-bounding-box)))
 
   ; Overloaded outputs from VisualNode
+  (output scene-updatable g/Any :cached (g/fnk [_node-id spine-data-handle spine-default-animation spine-skin spine-vertex-buffer]
+                                            (produce-spine-updatable _node-id spine-data-handle spine-default-animation spine-skin spine-vertex-buffer)))
```

Comment:
... (continued from above) Replace the `(g/fnk ...)` clause with your `defnk` here:

```suggestion
  (output scene-updatable g/Any :cached produce-spine-updatable)
```